zulipterminal/api_types.py

```diff
@@ -21,8 +21,8 @@ class PrivateComposition(TypedDict):
 class StreamComposition(TypedDict):
     type: Literal["stream"]
     content: str
-    to: str  # stream name  # TODO: Migrate to using int (stream id)
-    subject: str  # TODO: Migrate to using topic
+    to: Optional[int]
+    topic: str
```

Comment:
As per a comment in the stream, it may be cleaner to separate the compose and message parts of the topic migration into separate commits (with associated test changes in each).